db-migrations/000007_adding_functions.down.sql

```diff
@@ -0,0 +1,10 @@
+-- schema changes to support dual writes
+BEGIN;
+
+DROP FUNCTION get_resource_by_arn_id(character varying, timestamp without time zone);
```

Comment:
NIT: adding "IF EXISTS" to drop  raises our chances of recovering from botched schema (i.e. somehow some of the functions are defined but others are not, and we want to roll back)

Reply:
Ah i missed it 👍